blazor/datagrid/column-template.md

```diff
@@ -81,6 +81,69 @@ N> The column template feature is used to render the customized element value in
 The following screenshot represents the column Template.
 ![Blazor DataGrid with Column template](./images/blazor-datagrid-column-template.png)
 
+## Render other components in a column
+
+It is possible to render the other components in a column using the column template feature of the Grid.
```

Comment:
It is possible to render other components in a column using the column template feature of the Grid.